### ISSUE TITLE
A somewhat unrelated question.一个稍显无关的问题

### ISSUE THREAD

**Issue by szh404** [MarsZhaoYT/Sar2Opt-Heterogeneous-Dataset#4]
Your work is excellent, and I've learned a lot from you. 
The quality of your dataset is also very high, and I've accomplished a lot based on your dataset. Thank you ：)

I'd like to ask, for SAR images in SAR2Opt dataset, if I want to perform denoising, edge detection, texture extraction, and so on, are there any recommended methods? If so, could you please let me know?

**Comment by MarsZhaoYT** [comment]
Glad to hear our work is useful to your research. 

For SAR image denoising, if utilized in pre-processing, we usually use some simple traditional image processing methods, such as mean-value filter, median filter etc. The edge and texture detection can be handled with Sobel operator or Laplacian operator. 

In addition, there have been many deep learning-based methods for SAR image processing, especially diffusion-based methods. But I haven't tried these algorithms yet... : p

**Comment by szh404** [comment]
> Glad to hear our work is useful to your research.
> 
> For SAR image denoising, if utilized in pre-processing, we usually use some simple traditional image processing methods, such as mean-value filter, median filter etc. The edge and texture detection can be handled with Sobel operator or Laplacian operator.
> 
> In addition, there have been many deep learning-based methods for SAR image processing, especially diffusion-based methods. But I haven't tried these algorithms yet... : p

Thank you for your reply! Your advice is useful and I will consider it. 
By the way, Are you still studying on this? And I feel a little confused about the training of GAN. Can you share some useful hyperparameter you thought, or you have shown them in this repo code :>? 
Best wishes :) Happy everyday and live smooth!

**Comment by MarsZhaoYT** [comment]
Yes u r right, the convergence of GAN model is truly a bit difficult.  In the comparison, we followed the most of the parameter settings of official implementations to maintain the generation quality in the comparison. You can check the settings in '../option/train_options.py' or the parsers in 'main.py'. 

And unfortunately, I'm not working on the image translation anymore, instead I'm moving on to Change Detection and Self-supervised Learning in RS images... 
Best wishes :)

**Comment by szh404** [comment]
> Yes u r right, the convergence of GAN model is truly a bit difficult. In the comparison, we followed the most of the parameter settings of official implementations to maintain the generation quality in the comparison. You can check the settings in '../option/train_options.py' or the parsers in 'main.py'.
> 
> And unfortunately, I'm not working on the image translation anymore, instead I'm moving on to Change Detection and Self-supervised Learning in RS images... Best wishes :)

Thank you for your replying! Hope you will make more achievements! 
Best wishes! :)
